yarn-project/end-to-end/scripts/network_test.sh

```diff
@@ -18,6 +18,7 @@ TEST="$1"
 
 REPO=$(git rev-parse --show-toplevel)
 if [ "$(uname)" = "Linux" ] && [ "$(uname -m)" = "x86_64" ]; then
+  # Use a lock so this can be ran in parallel
```

Comment:
i could be missing something, where is this lock?

Reply:
old comment woops